css_tasks/task11/scss/_general.scss

```diff
@@ -0,0 +1,104 @@
+@import "variables";
+@import "mixins";
+
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+  font-family: $primaryFont;
+  color: $mainTextColor;
+}
+.container {
+  padding: 0 15%;
+}
+.container-standard {
+  padding: 0 14%;
+}
+picture {
+  display: block;
+}
+.main-header {
+  padding-top: $standardTopPadding;
+  &__headline {
+    @include decorativeText;
+    text-align: center;
+  }
+  &__title {
+    position: relative;
+    padding-top: 1rem;
+    font-family: $secondaryFont;
+    font-weight: bold;
+    font-size: $titleFontSize;
+    color: $secondaryTextColor;
+    text-transform: uppercase;
+    text-align: center;
+  }
+  &__separator {
+    @include separator;
+  }
+  &__text {
+    margin: 5rem auto 0;
+    width: 80%;
+    @include normalText;
+  }
+}
+.main-header--design,
+.main-header--clients {
+  padding-top: 3rem;
```

Comment:
this code looks too strange, what is a reason to make it absolute

Reply:
There is picture in design and clients section. Picture is added in `picture` tag, and second background  ( grey color) ias added in ::after. I need these headers to be positioned absolutely because in other case layout will be broken ( check in devTools). I could ,of course, add picture  in css with `background` property,  and second background also in css. In this case I wouldn't have to position headers absolutely. Am I missing something ? :)